packages/messages/src/browser/style/notifications.css

```diff
@@ -15,188 +15,188 @@
 
 .theia-Notification .buttons {
-    display: flex;
-    flex-direction: row;
-    order: 4;
-    white-space: nowrap;
-    align-self: flex-end;
-    height: 40px;
+  display: flex;
+  flex-direction: row;
+  order: 4;
+  white-space: nowrap;
 }
 
 .theia-Notification .buttons > button {
-    background-color: var(--theia-ui-button-color);
-    color: var(--theia-ui-button-font-color);
-    border: none;
-    border-radius: 0px;
-    text-align: center;
-    text-decoration: none;
-    display: inline-block;
-    padding: 0px 10px;
-    margin: 0px;
-    font-size: var(--theia-ui-font-size1);
-    outline: none;
-    cursor: pointer;
+  font-size: var(--theia-ui-font-size1);
+  display: inline-block;
+  margin: 0;
+  padding: 0 10px;
+  cursor: pointer;
+  text-align: center;
+  text-decoration: none;
+  color: var(--theia-ui-button-font-color);
+  border: none;
+  border-radius: 0;
+  outline: none;
+  background-color: var(--theia-ui-button-color);
 }
 
 .theia-Notification .buttons > button:hover {
-    background-color: var(--theia-accent-color1);
+  background-color: var(--theia-accent-color1);
 }
 
 @-moz-keyframes fadeInDown {
-    from {
-        opacity: 0;
-        -webkit-transform: translate3d(0, -100%, 0);
-        transform: translate3d(0, -100%, 0);
-    }
-    to {
-        opacity: 1;
-        -webkit-transform: none;
-        transform: none;
-    }
+  from {
+    -webkit-transform: translate3d(0, -100%, 0);
+            transform: translate3d(0, -100%, 0);
+    opacity: 0;
+  }
+  to {
+    -webkit-transform: none;
+            transform: none;
+    opacity: 1;
+  }
 }
 
 @-webkit-keyframes fadeInDown {
-    from {
-        opacity: 0;
-        -webkit-transform: translate3d(0, -100%, 0);
-        transform: translate3d(0, -100%, 0);
-    }
-    to {
-        opacity: 1;
-        -webkit-transform: none;
-        transform: none;
-    }
+  from {
+    -webkit-transform: translate3d(0, -100%, 0);
+            transform: translate3d(0, -100%, 0);
+    opacity: 0;
+  }
+  to {
+    -webkit-transform: none;
+            transform: none;
+    opacity: 1;
+  }
 }
 
 @-o-keyframes fadeInDown {
-    from {
-        opacity: 0;
-        -webkit-transform: translate3d(0, -100%, 0);
-        transform: translate3d(0, -100%, 0);
-    }
-    to {
-        opacity: 1;
-        -webkit-transform: none;
-        transform: none;
-    }
+  from {
+    -webkit-transform: translate3d(0, -100%, 0);
+            transform: translate3d(0, -100%, 0);
+    opacity: 0;
+  }
+  to {
+    -webkit-transform: none;
+            transform: none;
+    opacity: 1;
+  }
 }
 
 @keyframes fadeInDown {
-    from {
-        opacity: 0;
-        -webkit-transform: translate3d(0, -100%, 0);
-        transform: translate3d(0, -100%, 0);
-    }
-    to {
-        opacity: 1;
-        -webkit-transform: none;
-        transform: none;
-    }
+  from {
+    -webkit-transform: translate3d(0, -100%, 0);
+            transform: translate3d(0, -100%, 0);
+    opacity: 0;
```

Comment:
@BroKun thanks for the PR!

I agree with @akosyakov, even with ignoring whitespace changes it takes some time to spot the real changes.

Reply:
thanks for the cleanup!